docs/running-clusterfuzzlite/google_cloud_build.md

```diff
@@ -187,29 +187,45 @@ steps:
     env:
       - 'CLOUD_BUCKET=<your-cloud-bucket>'
       - 'REPOSITORY=<your-repo-name>'
-      - 'LANGUAGE=c++'
+      - 'LANGUAGE=c++' # Change this to your project's language.
       - 'SANITIZER=coverage'
       - 'CFL_PLATFORM=gcb'
-      - 'FILESTORE=gsutil'
 
   - name: gcr.io/oss-fuzz-base/clusterfuzzlite-run-fuzzers:v1
     env:
       - 'CLOUD_BUCKET=<your-cloud-bucket>'
       - 'REPOSITORY=<your-repo-name>'
-      - 'LANGUAGE=c++'
+      - 'LANGUAGE=c++' # Change this to your project's language.
       - 'MODE=coverage
       - 'SANITIZER=coverage'
       - 'CFL_PLATFORM=gcb'
-      - 'FILESTORE=gsutil'
 ```
 {% endraw %}
 
-To run this workflow periodically: set up a trigger that runs this workflow *manually*.
+To run this workflow periodically: set up a trigger that runs this workflow
+*manually*.
 Then follow the
 [documentation](https://cloud.google.com/build/docs/automating-builds/create-scheduled-triggers)
 on making the workflow run on a schedule.
 We recommend scheduling coverage reports to run once a day.
 
+## Viewing crashes
+
+Google Cloud Build does not offer an easy way to download files associated with
+a build. Therefore, to download crashes that were found by ClusterFuzzLite,
+inspect the logs to find the name of the crash file and then download it
+from `<your-cloud-bucket>/crashes/<fuzzer>/<sanitizer>/<crash-file>`
```

Comment:
Provide a concrete example of this? Also can we have an explicit gs:// scheme prefix here in the example (i.e.  `gs://<your-cloud-bucket>/...`), rather than a `<your-cloud-bucket-without-gs>` later? gs:// isn't really part of the "cloud bucket name" and is really a scheme specific to gsutil.